build.gradle

```diff
@@ -360,11 +366,11 @@ task dbLoad_mssql(dependsOn: 'compileJava', type: JavaExec) {
 	args = ["dbScript", "-S", "insertBaseData-mssql.sql", anetConfig]
 }
 
-task dbLoad_psql(dependsOn: "dbLoad_psql_createData", type: com.bmuschko.gradle.docker.tasks.container.DockerExecContainer) {
+task dbLoad_psql(dependsOn: "pushSqlBaseData", type: DockerExecContainer) {
 	group = "database"
 	description = "Runs the ANET database load command for PostgreSQL; loads demo data."
-	targetContainerId { dbContainerName }
-	commands = [ ["psql", "-U", run.environment["ANET_DB_USERNAME"], "-d", run.environment["ANET_DB_NAME"], "-f", "/hostdata/insertBaseData-psql.sql"] as String[] ]
+	containerId = dbContainerName
+	commands = [ ["psql", "-U", run.environment["ANET_DB_USERNAME"], "-d", run.environment["ANET_DB_NAME"], "-f", containerPushDataPath + "/insertBaseData-psql.sql"] as String[] ]
```

Comment:
Prefer this instead of string concatenation:
```
	commands = [ ["psql", "-U", run.environment["ANET_DB_USERNAME"], "-d", run.environment["ANET_DB_NAME"], "-f", "${containerPushDataPath}/insertBaseData-psql.sql"] as String[] ]
```

Reply:
I agree, but let's to this consistent them. I will replace all `+` string contatenations.